qa/rpc-tests/wallet_sendmany_any_taddr.py

```diff
@@ -112,7 +112,7 @@ def run_test(self):
             'ANY_TADDR',
             [{'address': recipient, 'amount': 20}],
             1, DEFAULT_FEE, 'AllowRevealedSenders')
-        wait_and_assert_operationid_status(self.nodes[3], myopid, "failed", "Insufficient funds: have 14.99998, need 20.00001; note that coinbase outputs will not be selected if you specify ANY_TADDR or if any transparent recipients are included.")
+        wait_and_assert_operationid_status(self.nodes[3], myopid, "failed", "Insufficient funds: have 14.99998, need 20.00001; note that coinbase outputs will not be selected if you specify ANY_TADDR.")
```

Comment:
The new message is just finer-grained, but I think it does need some rephrasing.

The gist is that if there were transparent recipients on this tx, it would also include the “if any transparent recipients are included” bit.

What it should probably say is something like
* “note that coinbase outputs were not selected because you specified ANY_TADDR”,
* “note that coinbase outputs were not selected because you included transparent recipients”, or
* “note that coinbase outputs were not selected because you specified ANY_TADDR and included transparent recipients”

depending on the content of the request. There are also two other cases that come up:
* “coinbase outputs will not be selected if the `privacyPolicy` parameter is not set to `AllowRevealedSenders` or weaker” and
* “may require selecting transparent coins that were sent to multiple Unified Addresses, which is not enabled by default […] `AllowLinkingAccountAddresses` or weaker […]”.

Reply:
This has been updated with the unified coinbase handling.